doc: drop minimum waiting time as hard guideline (#33879)
nodejs/node: onboarding.md

  reviewers. If you are leaving comments about issues that could be identified
  by tools but are not, consider implementing the necessary tooling.
* Minimum wait for comments time
  * Node.js is a distributed project, and it is important that other people
    have a chance to weigh in on changes that affect them. We do not enforce

>     a fixed waiting time, and pull requests that can generally be expected

Reviewer: ```suggestion
    a fixed waiting time. Pull requests that are expected
```

>     not to be contentious can be merged as soon as they have approvals and a

Reviewer: ```suggestion
    to be uncontentious can be merged as soon as they have approvals and a
```

    -1`
* Collaborators are in alphabetical order by GitHub username.
* Optionally, include your personal pronouns.

> * Label your pull request with the `doc`, `notable-change` labels.

Reviewer: ```suggestion
* Label your pull request with the `doc` and `notable-change` labels.
```
